csm_web/frontend/static/frontend/css/style.css

```diff
@@ -11,6 +11,10 @@ body {
 	overflow-x: hidden;
 }
 
+button:hover {
```

Comment:
These CSS edits should also be moved to a new CSS file (created again in the `resource-aggregation-tabs` branch).